distributed/comm/inproc.py

```diff
@@ -30,14 +29,10 @@ class Manager:
     def __init__(self):
         self.listeners = weakref.WeakValueDictionary()
         self.addr_suffixes = itertools.count(1)
-        with warnings.catch_warnings():
-            # Avoid immediate warning for unreachable network
-            # (will still warn for other get_ip() calls when actually used)
```

Comment:
this code is run at import:
https://github.com/dask/distributed/blob/713204b9757065cc1ce6df3574957a79a9f56ea6/distributed/comm/inproc.py#L76

however it's memoized so because the warning is skipped the first time it's skipped on all subsequent calls: https://github.com/dask/distributed/blob/713204b9757065cc1ce6df3574957a79a9f56ea6/distributed/utils.py#L125